src/Symfony/Component/Validator/Constraints/IsNull.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @api
+ */
+class IsNull extends Constraint
+{
+    public $message = 'This value should be null.';
+}
+
+if (PHP_VERSION_ID < 70000) {
+  // Null is a reserved word in PHP7.
+  class_alias('Symfony\Component\Validator\Constraints\IsNull', 'Symfony\Component\Validator\Constraints\Null');
```

Comment:
does it need to be in this file ? couldn't it be in the ``Null.php`` file instead ?